frontend/components/organiser/EventDrilldownSidePanel.tsx

```
@@ -48,16 +44,14 @@ const EventDrilldownSidePanel = ({
-            } hover:bg-white px-8 py-3 transition ease-in-out`}
-            onClick={() => setCurrSidebarPage("Forms")}
-          >
-            Forms
-          </div>
-        )}
+        <div
+          className={`text-organiser-dark-gray-text font-bold text-md ${
+            currSidebarPage === "Forms" ? "bg-white" : ""
+          } hover:bg-white px-8 py-3 transition ease-in-out`}
+          onClick={() => setCurrSidebarPage("Forms")}
+        >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)